DocumentDBStudio/Nodes/TreeNodes.cs

```diff
@@ -17,6 +19,8 @@ internal class AccountSettings
         /// Specifies whether this account settings should be persisted locally  
         /// </summary>
         public bool IsPersistedLocally;
+        public List<string> collectionTokens;
```

Comment:
Nit. Completely forgetting how you're storing and passing these tokens, but could you store these in one structure, Dictionary<Collection, CollectionToken> or similar like List<KeyValuePari<Collection, CollectionToken>